build.sh

```diff
@@ -108,7 +108,7 @@ ${AOM_TEST}/aomdec $ENC_FILE -o $DEC_FILE
 # Input/Output compare
 tail -n+2 $DEC_FILE > /tmp/dec_file
 tail -n+2 $REC_FILE > /tmp/rec_file
-cmp /tmp/dec_file /tmp/rec_file
+cmp /tmp/dec_file /tmp/rec_file || printf '\e[1;31m%-6s\e[m\n\n' "Desync detected!!!" && exit 1
```

Comment:
```suggestion
cmp /tmp/dec_file /tmp/rec_file || printf '\e[1;31m%-6s\e[m\n\n' 'Desync detected!!!' && exit 1
```

the ! could be problematic if you do not single-quote the string